api/mod/tests/conftest.py

```diff
@@ -30,3 +40,37 @@ def session() -> Generator:
     yield SETTINGS
 
     shutil.rmtree(SETTINGS.data_folder)
+
+
+@pytest.fixture(scope="session", autouse=True)
+def apply_migrations() -> Generator:
+    """Apply migrations to test database"""
+    config = Config(str(Path.cwd().joinpath("alembic.ini")))
+    config.set_main_option(
+        "script_location", str(Path.cwd().joinpath("mod").joinpath("alembic"))
+    )
+    config.set_main_option("sqlalchemy.url", SETTINGS.database_url)
+    alembic.command.upgrade(config, "head")
+    yield
+    alembic.command.downgrade(config, "base")
```

Comment:
Consider also seeding the database here with our `test_data.json`

Reply:
This could heavily streamline our integration tests (not having to create a test user each time, etc)

Reply:
Will save this for later (#86)